Lib/test/test_collections.py

```diff
@@ -9,6 +9,7 @@
 from random import choice, randrange
 from itertools import product, chain, combinations
 import string
+import array
```

Comment:
Nitpick: These imports are basically sorted alphabetically, so this line may move to the first line.

Reply:
done